engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/videoSettings/DynamicShadows.java

```diff
@@ -50,4 +50,8 @@ public static DynamicShadows find(boolean dynamicShadows, boolean pcf) {
         }
         return OFF;
     }
+
+    public String getDisplayName() {
```

Comment:
`displayName` probably does not reflect the nature of the string, maybe `langKey` or something like that would be better.